src/libcommon/utility/utility.cpp

```diff
@@ -75,6 +75,9 @@ namespace KDC {
 
 const int CommonUtility::logsPurgeRate = 7;               // days
 const int CommonUtility::logMaxSize = 500 * 1024 * 1024;  // MB
+#if defined(_WIN32)
+size_t CommonUtility::_maxPathWin = 0;
```

Comment:
You can define a static variable within the function `maxPathLength` itself. This way we don't create a variable the rest of the application should not be aware of (encapsulation).

Reply:
@herve-er
[define maxPathWin in maxPathLength()](https://github.com/Infomaniak/desktop-kDrive/pull/239/commits/b2b371c7f7e5c0cd809c1c438adfd2c98e035bc5)